Pod/Classes/ViewControllers/ARComponentViewController.m

```diff
@@ -87,6 +87,12 @@ - (void)viewWillAppear:(BOOL)animated
     NSMutableDictionary *appProperties = [self.rootView.appProperties mutableCopy];
     appProperties[@"isVisible"] = @YES;
     self.rootView.appProperties = appProperties;
+    
+    double delayInSeconds = 5.0;
```

Comment:
Curious where this 5.0 came from? (Was it spec'ed out, is it standard, etc.?)

Reply:
Oh that was a testing thing I'll remove it